README.md

```diff
@@ -82,6 +82,18 @@ A **Vagrant** configuration is included for development purposes. To use it, com
 - Run `vagrant ssh -c "cd /vagrant && foreman start"`
 - Open `http://mastodon.local` in your browser
 
+To set up **M1/2 MacOS** for native development, complete the following steps:
+
+- Install Ruby 3.2 (using [rbenv](https://github.com/rbenv/rbenv) for easy installation and management of ruby versions)
```

Comment:
I'd use "latest stable ruby" and might just say "use a ruby version manager" instead of rbenv specifically. I used `asdf` and everything is fine, for example.

Reply:
Agreed, will edit.